.code-samples.meilisearch.yaml

```diff
@@ -146,8 +271,81 @@ update_sortable_attributes_1: |-
 reset_sortable_attributes_1: |-
 search_parameter_guide_sort_1: |-
 geosearch_guide_filter_settings_1: |-
+  ```csharp
+  List<string> attributes = new() { "_geo" };
+  UpdateStatus result = await index.UpdateFilterableAttributes(attributes);
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet) 
 geosearch_guide_filter_usage_1: |-
+  ```csharp
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+  var index = client.Index("Restaurants");
+  
+  SearchQuery filters = new SearchQuery() { Filter = "_geoRadius(45.4628328, 9.1076931, 2000)" };
+  SearchResult<Restaurant> restaurants = await index.Search<Restaurant>("", filters);
+  foreach (var restaurant in restaurants.Hits)
+  {
+      Console.WriteLine(restaurant.Name);
+  }
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
 geosearch_guide_filter_usage_2: |-
+  ```csharp
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+  var index = client.Index("Restaurants");
+  
+  SearchQuery filters = new SearchQuery()
+  {
+      Filter = new string[] { "_geoRadius(45.4628328, 9.1076931, 2000) AND type = pizza" }
+  };
+  
+  SearchResult<Restaurant> restaurants = await index.Search<Restaurant>("restaurants", filters);
+  foreach (var restaurant in restaurants.Hits)
+  {
+      Console.WriteLine(restaurant.Name);
+  }
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
 geosearch_guide_sort_settings_1: |-
+  ```csharp
+  List<string> attributes = new() { "_geo" };
+  UpdateStatus result = await index.UpdateSortableAttributes(attributes);
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet) 
 geosearch_guide_sort_usage_1: |-
+  ```csharp
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+  var index = client.Index("Restaurants");
+  
+  SearchQuery filters = new SearchQuery()
+  {
+      Sort = new string[] { "_geoPoint(48.8583701,2.2922926):asc" }
+  };
+  
+  SearchResult<Restaurant> restaurants = await index.Search<Restaurant>("", filters);
+  foreach (var restaurant in restaurants.Hits)
+  {
+      Console.WriteLine(restaurant.Name);
+  }
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
 geosearch_guide_sort_usage_2: |-
+  ```csharp
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+  var index = client.Index("Restaurants");
+  
+  SearchQuery filters = new SearchQuery()
+  {
+      Sort = new string[] {
+            "_geoPoint(48.8583701,2.2922926):asc",
+            "rating:desc"
+      }
+  };
+  
+  SearchResult<Restaurant> restaurants = await index.Search<Restaurant>("restaurants", filters);
+  foreach (var restaurant in restaurants.Hits)
+  {
+      Console.WriteLine(restaurant.Name);
+  }
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
```

Comment:
Can you put a new line at the end of this file, please? :)
To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...) thanks to the .editorconfig file in this repo.
See more about editorconfig: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/meilisearch-dotnet/blob/main/.editorconfig